Library/Formula/castxml.rb

```diff
@@ -0,0 +1,24 @@
+class Castxml < Formula
+  desc "C-family Abstract Syntax Tree XML Output"
+  homepage "https://github.com/CastXML/CastXML"
+  url "http://http.debian.net/debian/pool/main/c/castxml/castxml_0.1+git20160202.orig.tar.xz"
```

Comment:
Please don't use a plaintext tarball for the url. Debian has enough secure mirrors to use one of 'em.